javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,244 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named transformToLis that, given an object, returns an array of the key value pairs as html list items.
+
+For example:
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj){
+  let returnArray = [];
+
+  for (const property in obj) {
+    returnArray.push(`<li>${property}: ${obj.name}</li>`);
+    returnArray.push(`<li>${property}: ${obj.age}</li>`);
+  }
+  returnArray.splice(1,1);
+  returnArray.splice(1,1);
+  //console.log(returnArray);
+  return returnArray;
+}; // Okay I know this probably wasn't how it was supposed to be solved, but it works ;v
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named count that, given an integer and an array of arrays, uses either filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+
+Note: You might need to use the same method more than once.
+
+For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+------------------------------------------------------------------------------------------------ */
+
+const count = (target, input) => {
+  
+  let count = 0;
+  
+   for(let i = 0; i <= input.length - 1; i++) {
+        for(let j = 0; j <= input[i].length - 1; j++){         
+           if (input[i][j] === target) {
+          count += 1;
+      }
+    }
+  }
+
+return count;
```

Comment:
Good work reaching a solution here too!

Another approach might be:
- to flatten this array, see method here: [flat()](https://dev.to/stlnick/javascript-array-flat-4l8f)
- When it is one dimensional, we can then use `reduce()` to count the number of times an integer is present
- With reduce we have our `acc` and our `curr` (the current item we are on at the time in iteration etc.`
- If the `curr === target`, then we need to increment count